src/server/scripts/Shadowlands/Sanctum of Domination/boss_sylvanas_windrunner.cpp

```diff
@@ -0,0 +1,3691 @@
+{
+    3, 3, 4
+};
+
+Position const SylvanasIntro[4] =
+{
+    { 231.15799f, -832.816f,   4105.0386f          },
+    { 242.00348f, -840.51215f, 4105.0386f          },
+    { 241.23091f, -830.0955f,  4105.0386f          },
+    { 225.73611f, -844.0746f,  4104.9882f, 1.3613f }
+};
+
+Position const SylvanasVeilOfDarknessPos = { 255.0392f, -824.69995f, 4205.122f };
+
+Position const DesecratingShotArrowShape[5] =
+{
+    { -0.43609f,  4.98095f, 4105.0386f  },
+    {  2.38811f,  4.82665f, 4105.0386f  },
+    {  5.21233f,  4.67245f, 4105.0386f  },
+    {  5.05808f,  1.84821f, 4105.0386f  },
+    {  4.90381f, -0.97601f, 4105.0386f  }
+};
+
+Position const RiveThrowPos[8] =
+{
+    { 166.617f, -856.656f, 4113.2285f },
+    { 208.542f, -761.602f, 4113.2285f },
+    { 303.332f, -803.056f, 4113.2285f },
+    { 302.793f, -858.021f, 4113.2285f },
+    { 183.836f, -777.633f, 4113.2285f },
+    { 206.884f, -897.526f, 4113.2285f },
+    { 261.473f, -898.365f, 4113.2285f },
+    { 287.948f, -778.238f, 4113.2285f }
+};
+
+Position const RiveFinish =            { 235.1163f, -829.90106f, 4105.0386f, 5.4824f   };
+
+Position const SylvanasPhase2Pre =     { -14.5625f, -943.441f,   4999.990f,  0.8928f   };
+
+Position const SylvanasPhase3Pre =     { -258.991f, -1265.9966f, 5667.114f,  0.3118f   };
+
+Position const SylvanasPlaform1Pos =   { -280.646f, -1245.48f,   5672.13f,   2.3046f   };
+
+Position const PlaformsMiddlePos[4] =
+{
+    { -289.503f, -1237.0555f, 5671.9053f, 5.5017f },
+    { 0.f },
+    { 0.f },
+    { 0.f }
+};
+
+Position const SylvanasUnconciousPos = { -249.876f, -1252.4791f, 5667.1157f, 3.3742f   };
+
+// Sylvanas Windrunner - 175732
+struct boss_sylvanas_windrunner : public BossAI
+{
+    boss_sylvanas_windrunner(Creature* creature) : BossAI(creature, DATA_SYLVANAS_WINDRUNNER), _windrunnerActive(false), _sayDaggersOnCD(false),
+        _sayDesecratingOnCD(false), _lastSpellUsed(0), _windrunnerCastTimes(0), _riveCastTimes(0) { }
+
+    void JustAppeared() override
+    {
+        _instance = me->GetInstanceScript();
+
+        scheduler.ClearValidator();
+
+        // HACKFIX: for some reason, SPELL_SYLVANAS_DISPLAY_POWER_SUFFERING does not override her DisplayPowerId
+        me->SetOverrideDisplayPowerId(DATA_OVERRIDE_DISPLAY_POWER_SUFFERING);
+
+        if (Creature* ridingCopy = me->FindNearestCreature(NPC_SYLVANAS_SHADOW_COPY_RIDING, 10.0f, true))
+        {
+            ridingCopy->SetReactState(REACT_PASSIVE);
+
+            ridingCopy->SetUnitFlags(UNIT_FLAG_IMMUNE_TO_PC);
+            ridingCopy->SetUnitFlags(UNIT_FLAG_IMMUNE_TO_NPC);
+            ridingCopy->SetUnitFlags(UNIT_FLAG_NOT_SELECTABLE);
+        }
+    }
+
+    void EnterEvadeMode(EvadeReason /*why*/) override
+    {
+        Talk(SAY_DISENGAGE);
+
+        _EnterEvadeMode();
+        summons.DespawnAll();
+        _instance->SendEncounterUnit(ENCOUNTER_FRAME_DISENGAGE, me);
+        _instance->SetBossState(DATA_SYLVANAS_WINDRUNNER, FAIL);
```

Comment:
_DespawnAtEvade will do that already on its own

Reply:
fixed on next commit.